test/env_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+describe 'Lotus' do
+  before do
+    ENV['LOTUS_ENV'] = 'test'
+  end
+
+  describe '.env' do
+    it 'returns env object' do
+      Lotus.env.must_be_kind_of String
+    end
+
+    it 'return "test"' do
+      Lotus.env.must_equal 'test'
+    end
+  end
+
+  describe '.env?(:test)' do
```

Comment:
what do you think about changing it to `.env?` and adding a context line next to it `when environment is matched`?